qemu/tests/block_stream.py

```diff
@@ -11,16 +12,26 @@
 class BlockStreamTest(blk_stream.BlockStream):
 
     def get_image_size(self, image_file):
-        qemu_img = utils_misc.get_qemu_img_binary(self.params)
-        cmd = "%s info %s" % (qemu_img, image_file)
-        info = utils.system_output(cmd)
-        size = re.findall("(\d+) bytes", info)
+        try:
+            qemu_img = utils_misc.get_qemu_img_binary(self.params)
+            cmd = "%s info %s" % (qemu_img, image_file)
+            info = process.system_output(cmd)
+            size = int(re.findall("(\d+) bytes", info)[0])
+        except process.CmdError, err:
+            if "another process using the image" in str(err):
```

Comment:
with `proess.CmdError` , `err`  is `Command '/bin/qemu-img info /home/kvm_autotest_root/images/rhel75-64-virtio-scsi.qcow2' failed (rc=1)` , doesn't include the given message `another process using the image` .
I think we can remove this check,  when get size from qemu-img failed,  try to get it use qemu monitor.

Reply:
If qemu block layer change again, the output may change too.  Use fixed string is unreasonable.

Reply:
@yanan-fu Yes, updated, thanks.